renovate.json

```diff
@@ -1,4 +1,9 @@
 {
   "extends": ["config:base", ":semanticCommits"],
-  "schedule": "every weekend"
```

Comment:
Can we also please keep the schedule configuration for major updates?

Reply:
I think the major change of 3rd party libs  will not frequently。 the `weekend` `schedule` is useless for major changes.

If we put the "schedule": "every weekend", it will only run the check job on weekend.

Reply:
Makes sense.